main.py

```diff
@@ -35,14 +41,20 @@ def get_api():
 def get_landmarks(id):
     try:
         landmarks = Landmark.query.filter(Landmark.celestial_body_id == id)
-        return jsonify({'data': [e.serialize() for e in landmarks]})
+        if CelestialBodies.query.get(id) is not None:
+          return jsonify({'data': [e.serialize() for e in landmarks]})
+        else:
+          return jsonify({'errors': f'No celestial body with id: {id}.'}), 404
```

Comment:
Again, I added sad paths for the most common error I could see which is requesting landmarks for a nonexistent celestial body. If no landmarks exist it simply returns an empty list.